express-101/src/index.js

```diff
@@ -1,9 +1,13 @@
 const express = require('express');
 
 const app = express();
+const port = 3000;
 
-// 1) Create Routes
+app.get('/', (req, res) => res.send('Hey, I am server response'));
+app.get('/movies/:title', (req, res) => {
+  res.send(`My favorite movie is ${req.params.title}`);
```

Comment:
I think it would be better to use object destructuring here like ` const { title } = req.params;`